client/state/data-layer/wpcom/read/following/mine/index.js

```diff
@@ -19,6 +19,19 @@ import { errorNotice } from 'state/notices/actions';
 const ITEMS_PER_PAGE = 200;
 const MAX_ITEMS = 2000;
 
+export function toValidId( val ) {
```

Comment:
my issue with dumping grounds like utils is that you end up with deps that look like this:

```
/**
 * External Dependencies
 */
import url from 'url';
import page from 'page';
import { every } from 'lodash';

/**
 * Internal Dependencies
 */
import i18n from 'i18n-calypso';
import { state as SiteState } from 'lib/reader-site-store/constants';
import FeedDisplayHelper from 'reader/lib/feed-display-helper';
import PostStore from 'lib/feed-post-store';
import XPostHelper, { isXPost } from 'reader/xpost-helper';
import { setLastStoreId } from 'reader/controller-helper';
import { fillGap } from 'lib/feed-stream-store/actions';
import { recordAction, recordGaEvent, recordTrack } from 'reader/stats';
```

So then the data-layer, which is part of the redux stuff, which is part of build, now requires all that junk too.

I think I'd rather have a utils folder and break up the utils by thing to help keep the dependencies to a minimum?